.github/workflows/_tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Run tests
+on:
+  push:
+    branches: [ "*" ]
+env:
+  ER_USERNAME: ${{ secrets.ER_USERNAME }}
+  ER_PASSWORD: ${{ secrets.ER_PASSWORD }}
+  GCP_DATAFLOW_SERVICE_KEY: ${{ secrets.GCP_DATAFLOW_SERVICE_KEY }}
```

Comment:
```suggestion
  GEE_SERVICE_ACCOUNT_KEY: ${{ secrets.GEE_SERVICE_ACCOUNT_KEY }}
```

Let's rename this env variable and the corresponding secret to something more descriptive. If this is only used for Google Earth Engine (which I think it is), then maybe `GEE_SERVICE_ACCOUNT_KEY`?

(For others reading this, our reference for this workflow was a prior workflow I'd worked on for a previous job, in which we were deploying to GCP Dataflow from GitHub. But we're not doing that here, so the naming should reflect what we're actually doing.)

Also, @davidovichmarcos, how did you generate the secret used here? Is it in fact a service account or is this key tied to your personal user account? If the latter, let's make an actual service account for this.

Reply:
There's an actual service account for this configured in the repo settings already but it's stored as `EE_ACCOUNT` & `EE_PRIVATE_KEY_DATA` which _may_ be compatible here with some changes to the yaml.

Reply:
@davidovichmarcos I can share those credentials with for local testing

Reply:
I'm using this service account keys (which i created only for that purpose) but i can try setting a new one, and yes i'm using a real json file with valid credentials.

![Screenshot 2024-08-16 at 9 49 48 AM](https://github.com/user-attachments/assets/fdcdae7c-001e-4554-917d-deff245c5e13)